apps/platform-app/src/app/[locale]/(profile)/_actions/mutations/publishProfile.ts

```diff
@@ -23,9 +24,13 @@ export const publishProfile = withSentry(async (profileId: string) => {
     state: PublishingState.PENDING,
   })
 
-  await sendDiscordNotificationToModeratorChannel(
-    `User's **${updatedProfile.fullName}** profile has got new status: **${updatedProfile.state}**! [Show Profile](${process.env.NEXT_PUBLIC_APP_ORIGIN_URL}/moderation/profile/${updatedProfile.userId})`,
-  )
+  runEvaluateProfileAgent(foundProfile.userId)
```

Comment:
**nit:** i think we should add `await` here, so `await runEvaluateProfileAgent(foundProfile.id)`